### ISSUE TITLE
add links to API definitions in docs

### ISSUE THREAD

**Issue by DanAlbert** [android-ndk/ndk#787]
https://www.reddit.com/r/androiddev/comments/8vzqpq/android_ndk_glib_and_glibc_support/e5g99pe/

https://developer.android.com/ndk/guides/stable_apis ought to link directly to the version scripts that define each library so it's easy to see what is actually available and when.

**Comment by enh** [comment]
maybe. if someone can't understand from the headers (or the "Available since API level X" we've added to the API reference docs), i'm not sure they're going to be able to make much of the version scripts.

plus note that the reddit questioner wouldn't be helped by this. they're asking about two non-libc libraries.

**Comment by DanAlbert** [comment]
Getting from that page to the relevant headers isn't great either. Maybe we should start generating that page from the version scripts or something...

Tbh it's a good argument for actually adding the bionic docs to the API reference, but we'll need to find a way to do that without drowning out the rest.

**Comment by enh** [comment]
> Tbh it's a good argument for actually adding the bionic docs to the API reference, but we'll need to find a way to do that without drowning out the rest.

yeah, this is what i meant the other day about preferring the standard doxygen output to the google-ified doxygen output. the stuff i have in my tmp directory isn't too bad.

**Comment by NicciCz** [comment]
C
